src/App/mermaidData/syncApiDataIntoOfflineStorage/SyncApiDataIntoOfflineStorage.js

```diff
@@ -6,6 +6,54 @@ const SyncApiDataIntoOfflineStorage = class {
 
   _dexiePerUserDataInstance
 
+  _getTableNamesWhereSyncingWasRejectedListedPerProject =
```

Comment:
Ugh, the self documenting function names for manipulating data when there are no types to do the same thing better